test_file.py

```diff
@@ -0,0 +1,6 @@
+from c4v.classifier.classifier import ClassifierExperiment
+
+# branch name, experiment name
```

Comment:
Are we sure we want to put this file in the root folder?

Reply:
oh, I have to remove it, I push it just as a workaround in google colab, as I wasn't able to run it because of the python version